__tests__/app.test.js

```diff
@@ -0,0 +1,93 @@
+const app = require("../app.js")
+const db = require("../db/connection.js")
+const request = require("supertest")
+const seed = require("../db/seeds/seed.js")
+const data = require("../db/data/test-data")
+const endpoints = require("../endpoints.json")
+
+
+beforeEach(() => {
+    return seed(data)
+ })
+ 
+afterAll(() => {
+    return db.end()
+ })
+
+ describe("/api/topics", () => {
+    test("GET 200: responds with an array of topic objects, each with slug and description properties", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.length).toBe(3)
+            body.forEach((topic) => {
+                expect(topic).toHaveProperty("description")
+                expect(topic).toHaveProperty("slug")
+            })
+        })
+    })
+    test("GET 404: responds with a 404 not found error message when an incorrect path is given", () => {
```

Comment:
This isn't related to the /api/topics endpoint so I would put it in its own describe block to avoid confusion.

Reply:
Thanks for the feedback Rich, would you like me to implement these changes now?